src/components/Slider/Slider.js

```diff
@@ -16,13 +16,15 @@ const AnimatedOverlay = animated(StyledOverlay)
 const AnimatedSlider = animated(StyledSlider)
 
 export default function Slider({
+  initialSlide,
```

Comment:
Since we're adding a new prop, this should also be documented in the README.

This prop is also a good candidate to demonstrate in the example (as third slider)

Reply:
Added